enforcer/src/main/java/org/wso2/micro/gateway/enforcer/api/RequestContext.java

```diff
@@ -51,6 +52,7 @@
     private boolean clusterHeaderEnabled = false;
     //Denotes the specific headers which needs to be passed to response object
     private Map<String, String> responseHeaders;
+    private ArrayList<String> removeHeaders;
```

Comment:
This variable has no usages?

Reply:
Fixed with https://github.com/wso2/product-microgateway/pull/1785/commits/5e7db9dba2577979211d7713f6690403ca650326